analyze.py

```diff
@@ -0,0 +1,105 @@
+import sys
+import os
+def parseFile(fileName):
+    file = open(fileName)
+    suitesLines = []
+    suitesNames = []
+    totalLines = 0
+    res = {}
+    for i, line in enumerate(file.readlines()):
+        totalLines += 1
+        if(line.startswith("#")):
+            suitesLines.append(i + 1)
+            suitesNames.append(line)
+    suitesLines.append(totalLines)
+    file.seek(0)
+    lines = file.readlines()
+    for i in range(len(suitesNames)):

```

Comment:
Why not finish parse in one loop??

Reply:
Try in one loop but not convenient to collect result.